packages/protocol/test/governance/downtime_slasher_intervals.ts

```diff
@@ -0,0 +1,731 @@
+        // All the other block are good
+        await presetParentSealForBlocks(startBlock, slashableDowntime - 1, [
+          bitmapWithoutValidator[validatorIndexInEpoch],
+        ])
+        // last block with everything signed
+        await presetParentSealForBlocks(startBlock + slashableDowntime - 1, 1, [bitmapVI01])
+        const slotArrays = await calculateEverySlot(startBlock)
+        await assertRevert(
+          slasher.slash(
+            slotArrays.startSlots,
+            slotArrays.endSlots,
+            [validatorIndexInEpoch],
+            0,
+            [],
+            [],
+            [],
+            [],
+            [],
+            []
+          ),
+          'not down'
+        )
+      })
+      it('fails if one block in the middle was signed', async () => {
+        const startBlock = (epoch - 1) * epochBlockSize + 1
+
+        // All the other block are good
+        await presetParentSealForBlocks(startBlock, slashableDowntime, [
+          bitmapWithoutValidator[validatorIndexInEpoch],
+        ])
+        // middle block with everything signed
+        await presetParentSealForBlocks(startBlock + intervalSize, 1, [bitmapVI01])
+        const slotArrays = await calculateEverySlot(startBlock)
+        await assertRevert(
+          slasher.slash(
+            slotArrays.startSlots,
+            slotArrays.endSlots,
+            [validatorIndexInEpoch],
+            0,
+            [],
+            [],
+            [],
+            [],
+            [],
+            []
+          ),
+          'not down'
+        )
+      })
+      it('fails if the first block was signed using a big index', async () => {
+        await slasher.setNumberValidators(100)
+        await slasher.setEpochSigner(epoch, 99, validatorList[2])
+        const startBlock = (epoch - 1) * epochBlockSize + 1
+        // All the other block are good
+        await presetParentSealForBlocks(startBlock + 1, slashableDowntime - 1, [bitmapVI0])
+        // first block with everything signed
+        await presetParentSealForBlocks(startBlock, 1, [bitmapVI99])
+        const slotArrays = await calculateEverySlot(startBlock)
+        await assertRevert(
+          slasher.slash(
+            slotArrays.startSlots,
+            slotArrays.endSlots,
+            [99],
+            0,
+            [],
+            [],
+            [],
+            [],
+            [],
+            []
+          ),
+          'not down'
+        )
+      })
+      describe('when the validator was down', () => {
+        let startBlock: number
+        beforeEach(async () => {
+          startBlock = (epoch - 1) * epochBlockSize + 1
+          await presetParentSealForBlocks(startBlock, slashableDowntime, [
+            bitmapWithoutValidator[validatorIndexInEpoch],
+          ])
+        })
+        describe('when the slots cover the SlashableDowntime window', () => {
+          it('success if slots overlap', async () => {
```

Comment:
Wording nit: when possible, try to word these so it's a sentence, e.g. `it('succeeds when the slots overlap')`. The "fail" ones elsewhere are already good.